core/templates/dev/head/components/help_modal_directive.html

```diff
@@ -0,0 +1,60 @@
+<script type="text/ng-template" id="modals/userHelp">
+<div class="modal-body oppia-long-text protractor-test-welcome-modal">
+  <h1>Need Help with Something?</h1>
+  <p>
+    You can check out the resources in our help center or
+    take the tour of the exploration editor
+  </p>
+  <a href="https://oppia.github.io/#/" target="_blank" class="btn btn-success oppia-help-dismiss protractor-test-dismiss-welcome-modal" ng-click="goToHelpCenter()">Visit the Help Center</a>
+  <button class="btn btn-success oppia-welcome-start protractor-test-start-tutorial" ng-click="beginTutorial()">Take the Tour</button>
```

Comment:
Where is oppia-welcome-start defined? If it's not defined, let's remove the class from both here and /exploration_welcome_modal_directive.html.